.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: Report a bug in Container Builder
+labels: kind/bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! 
+        The preferred method to file a bug is [our JIRA board](https://issues.redhat.com/projects/KOGITO). You are still allowed to submit a GitHub issue, but you may experience delays in the feedback.
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the bug
+      description: >-
+        Describe the issue you are experiencing here to communicate to the
+        maintainers. Tell us what you were trying to do and what happened.
+        Provide a clear and concise description of what the problem is.
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: >-
+        Describe the expected behavior clearly and concisely.
+  - type: textarea
+    attributes:
+      label: Actual behavior
+      description: >-
+        Describe the actual behavior clearly and concisely.
+  - type: textarea
+    attributes:
+      label: How to Reproduce?
+      description: >-
+        Link to a small reproducer ((preferably a golang project that it is using this module)) or attach an archive containing the reproducer to the issue.
```

Comment:
we have to probably remove one level of parenthesis from (( to (

Reply:
fixed